images/openshift-applier/Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM centos:centos7
 
-ENV OC_CLIENT_MIRROR https://mirror.openshift.com/pub/openshift-v3/clients/3.7.26/linux/oc.tar.gz
+ENV OC_CLIENT_MIRROR https://mirror.openshift.com/pub/openshift-v3/clients/3.9.14/linux/oc.tar.gz
 
 USER root
 
 # Update System and install clients
 RUN  yum install -y epel-release; \
   yum install -y python-pip git; \
   pip install --upgrade pip;  \
-  pip install git+https://github.com/ansible/ansible.git@devel
+  pip install git+https://github.com/ansible/ansible.git@stable-2.5
```

Comment:
do we need to `pip install` ansible anymore? We just announced 2.5 GA, so there should be an RPM available.

Reply:
@etsauer had this discussion with @sherl0cks and he preferred the pip route to have alignment with https://github.com/rht-labs/labs-ci-cd/blob/master/docker/tool-box/Dockerfile#L13

My preference is also on RPM, but the RPM in EPEL is still on `2.4.x`.

Reply:
How about: [http://releases.ansible.com/ansible/rpm/release/epel-7-x86_64/](http://releases.ansible.com/ansible/rpm/release/epel-7-x86_64/)?